src/Std/Data/DHashMap/Lemmas.lean

```diff
@@ -181,19 +181,37 @@ theorem size_le_size_erase [EquivBEq α] [LawfulHashable α] {k : α} :
   Raw₀.size_le_size_erase ⟨m.1, _⟩ m.2
 
 @[simp]
+theorem fst_containsThenInsert {k : α} {v : β k} : (m.containsThenInsert k v).1 = m.contains k :=
+  Raw₀.containsThenInsert_fst _
+
+@[simp, deprecated fst_containsThenInsert (since := "2025-02-20")]
```

Comment:
The old name was correct! From the [naming convention](https://github.com/leanprover/lean4/blob/master/doc/std/naming.md#naming-algorithm-for-theorems-and-some-definitions):

> When encountering a structure projection, proceed as for normal functions, but put the name of the projection last.